### ISSUE TITLE
Video player enhancements

### ISSUE THREAD

**Issue by CodeCubeNeo** [hardcore-sushi/DroidFS#88]
Hello! First i want to thank you for this awesome app, that I and many other people use every day. It is a true pleasure. I would like to recommend some enhancements for the built-in video player, that wold remove the need for playing with an outside video player (like vlc). here are the enhancements in (my!) order of importance (first is most important):
- select/deselect/none audio tracks
- audio (and brightness) controls with swiping gestures (right half of the screen, up and down for lower/higher volume and the left half for lower/higher brightness)
- subtitles (select file from the encrypted volume)
If those were added it would be really awesome!!! Thank you for your awesome work!

**Comment by CodeCubeNeo** [comment]
I see that this is all dependent on the exo player so i will also post that there.

**Comment by CodeCubeNeo** [comment]
One another option is to also use a different video player altogether since ExoPlayer hasn't received any updates since November 2021 (and its from google). The best option would probably be a fork of ExoPlayer that is still meinatined (https://github.com/moneytoo/Player). Some other options I recommend are mpv for android (missing the no auto rotate feature), vlc for android (though tough to extract the actual video player) and I also found some weird ones like nova-player, QtAV...

**Comment by CodeCubeNeo** [comment]
Hello! I get your point, but If you could please just make gestures (optional as you said) and add an option to disable audio playing (not a mute button, but to actually disable output audio) that would be great and very well appreaciated.

**Comment by CodeCubeNeo** [comment]
Hello! I see that you have upgraded your media player and added some enhancements (thanks). But I am still really missing the gestures for brightness and volume and an option to disable the audio track (set audio track to none, not a 0% volume button). I would just really like to know, if those two things are being planned for someday and if so when?

**Comment by hardcore-sushi** [comment]
Yes, these features might be added in the future, but it's not the main priority for now, as they are extra features in the video player feature. I prefer to spend my time on the features that people generally expect from an encrypted volume app like stability, security, usability... But of course pull requests are welcome.

Also, why wouldn't a 0% volume button do the trick?

**Comment by CodeCubeNeo** [comment]
I get it, thanks for the quick response. The 0% volume would't work, because i have a music player in the backgroud (which does not require audio focus), so when i want to watch a video without sound and want to listen to music at the same time I can't do that.

**Comment by hardcore-sushi** [comment]
Yes Android is awesome.
